### PR TITLE
Merge pull request #1 from godsomex/config

### PR DESCRIPTION
**Pull request by Yommie00**
structure

### CODE REVIEW

**Review by godsomex — COMMENTED** [review]
please add this file to .gitignore. the vscode settings should be personal

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (godsomex) [inline .vscode/settings.json]: please add this file to the .gitignore

File: index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
```
Comment (godsomex) [inline index.html]: rename this file product-description.html

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,212 @@
+*{
```
Comment (godsomex) [inline styles.css]: product-des.css

File: .vscode/settings.json (hunk 1 above)
Comment (Yommie00) [reply .vscode/settings.json]: Alright. Thank you

File: styles.css (hunk 3 above)
Comment (Yommie00) [reply styles.css]: done

**Pull request closed without merging**